types/jsreport/index.d.ts

```diff
@@ -0,0 +1,14 @@
+// Type definitions for jsreport 2.9
+// Project: https://github.com/jsreport/jsreport
+// Definitions by: pofider <https://github.com/jsreport>
```

Comment:
Is the URL a typo?

Even if it's an org that you belong to, it's better to have the URLs of the actual contributors here.

Reply:
Yes, I fixed it with link to my profile https://github.com/pofider